packages/qbf/qbf.0.3/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+maintainer: "Simon Cruanes <simon.cruanes@inria.fr>"
+authors: [ "Simon Cruanes" ]
+license: "BSD-3-clause"
+homepage: "https://github.com/c-cube/ocaml-qbf"
+dev-repo: "git+https://github.com/c-cube/ocaml-qbf.git"
+bug-reports: "https://github.com/c-cube/ocaml-qbf/issues"
+tags: [ "clib:quantor" "clib:qdpll" "clib:picosat"  ]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "build" "@doc" "-p" name ] {with-doc}
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test & ocaml:version < "4.11"}
```

Comment:
What's the reason behind not testing on 4.11?

Reply:
Not sure, @c-cube might know more about that 😅

Reply:
ok, let's try. I'll remove my change if there is an issue

Reply:
It seems to be passing without any issues. Could you return the fix upstream?

Reply:
Will do!